build/azure-pipelines/common/computeNodeModulesCacheKey.ts

```diff
@@ -0,0 +1,51 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
how about adding a `sql-` prefix to the file name? make it easy to tell it is created by us